### ISSUE TITLE
More authors in copyright notice?

### ISSUE THREAD

**Issue by jl-wynen** [scipp/copier_template#218]
Our copyright notice (and package metadata) lists 'Scipp contributors' as authors with a link to the Scipp GH org. The org only shows the core devs. This means that other contributors are not directly visible. This could be an issue esp. for ESS* packages.

Should we change the copyright to list the specific package / repo instead of the org? Or is there an other way to clearly include everyone?

**Comment by nvaytet** [comment]
Maybe each project could have it different? In the copier template, we could have ESSsans contributors, that would point to https://github.com/scipp/esssans/graphs/contributors ?

**Comment by SimonHeybrock** [comment]
Or link to a page in the docs such as https://scipp.github.io/esssans/developer/index.html, where contributors can be added?

**Comment by nvaytet** [comment]
I guess copyright holder is ESS for ESS employees but this being an open source project what do we do if we have external contributors?

**Comment by jl-wynen** [comment]
They hold the copyright for their contributions.

As I understand it, if we use (with the correct package inserted)
```
# Copyright (c) 2024 $PACKAGE contributors
```
then this means that every contributor, or their employer, holds the copyright to their contributions.

I _think_ that if we use the organisation instead of the package in the copyright notice, we don't include all contributors. But we should.

**Comment by nvaytet** [comment]
Doesn't this go back to an old discussion we had about whether we even need the copyright notice in every file?
We should keep the license identifier, but can we just remove the copyright (and also the annoying year that comes with it?)

**Comment by jl-wynen** [comment]
That only solves part of the problem. We still have to name authors in the license file.

**Comment by nvaytet** [comment]
> That only solves part of the problem. We still have to name authors in the license file.

True :sweat_smile:

**Comment by jl-wynen** [comment]
They don't They have the copyright of that line. But we can't reasonable track this with comments. This is Git's job.

**Comment by jl-wynen** [comment]
Agree. My point is to move away from listing the _organisation_ members as copyright holders.

**Comment by nvaytet** [comment]
Can we remove the copyright notice from the files and just have it in the license, so that we don't have to constantly update the years in the files?
